api-docs/components/parameters/scim.json

```diff
@@ -0,0 +1,29 @@
+{
+    "filter": {
+      "name": "filter",
+      "in": "query",
+      "required": false,
+      "description": "A SCIM filter expression. The only operator currently supported is `eq`.",
+      "schema": {
+        "type": "string"
+      }
+    },
+    "startIndex": {
```

Comment:
in-case anyone is wondering, SCIM is a public standard which is why we can't use our usual pagination